quiche/src/tls.rs

```diff
@@ -1106,6 +1106,7 @@ extern fn keylog(ssl: *mut SSL, line: *const c_char) {
         full_line.push(b'\n');
 
         keylog.write_all(&full_line[..]).ok();
+        keylog.flush().ok();
```

Comment:
This is specifically required for the FFI layer because we are wrapping a `File` into a `BufWriter` which causes these writes not to be written out immediately without this `flush`.